src/components/quickSwitch/QuickSwitch.styl

```diff
@@ -5,12 +5,15 @@
     box-shadow: 0 0 0 5000px rgba(0, 0, 0, 0.8)
     transition: opacity 0.3s ease
     opacity: 0
+    content: ''
 
   &:hover
     z-index: 50
 
     &:before
+      display: table;
       opacity: 1
+      content: ''
```

Comment:
Are we using this as a flexbox alternative?

Reply:
I added the `content` property here to fix the problem where the `before` was not being rendered after removing the bootstrap. This makes `before` legible to be presented.